src/ci/docker/scripts/fuchsia-test-runner.py

```diff
@@ -502,6 +451,19 @@ def run(self, args):
 
         bin_path = os.path.abspath(args.bin_path)
 
+        # Find libstd and libtest
+        libstd_paths = glob.glob(os.path.join(self.rustlibs_dir(), "libstd-*.so"))
+        libtest_paths = glob.glob(os.path.join(self.rustlibs_dir(), "libtest-*.so"))
+
+        if not libstd_paths:
+            raise Exception(f"Failed to locate libstd (in {self.rustlibs_dir()})")
+
+        if not libtest_paths:
+            raise Exception(f"Failed to locate libtest (in {self.rustlibs_dir()})")
+
+        libstd_name = os.path.basename(libstd_paths[0])
+        libtest_name = os.path.basename(libtest_paths[0])
+
```

Comment:
What about pretending the libs path here so the template doesn't have to also know about it?